ethcore/private-tx/src/lib.rs

```diff
@@ -690,8 +701,8 @@ impl ChainNotify for Provider {
 	fn new_blocks(&self, imported: Vec<H256>, _invalid: Vec<H256>, _route: ChainRoute, _sealed: Vec<H256>, _proposed: Vec<Bytes>, _duration: Duration) {
```

Comment:
Out-of-scope for this PR:

Would be better to take the function parameters as slices to support more types than `Vec`, i.e, to use instead ```&[H256]```! 

But because this is part of a trait that would probably demand some work!